pkg/logutil/log.go

```diff
@@ -254,13 +254,14 @@ func (lg *wrapLogrus) V(l int) bool {
 	return int(lg.Logger.Level) <= logrusLevel
 }
 
-var once sync.Once
+var (
+	initLoggerOnce sync.Once
+	initLoggerErr  error
```

Comment:
Why change it to a global variable?

Reply:
This function will only be executed once. If an error occurs in the first execution, the next execution should continue to return err instead of nil.